config/rbac/secrets_role_binding.yaml

```
@@ -0,0 +1,13 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: openstack-provider-manager-secrets-rolebinding
```

Please drop redundant `rolebinding` from the name
```suggestion
  name: openstack-provider-manager-secrets
```